test/spec/storageSrv/storageFirebaseAdapterSrv.spec.js

```diff
@@ -0,0 +1,89 @@
+describe('testing service "storageFirebaseAdapter":', function () {
+    'use strict';
+
+    beforeEach(function(){
+        window.Firebase = function(path){
```

Comment:
Currentlymy mock is doing its job, if extra mocking will be needed than we can use the mockFirebase.